tests/test_baker.py

```diff
@@ -620,8 +623,9 @@ def test_create_model_with_valid_ips(self):
         assert DummyGenericIPAddressFieldForm(form_data).is_valid()
 
 
-@pytest.mark.django_db
-class TestBakerAllowsSaveParameters:
+class TestBakerAllowsSaveParameters(TestCase):
```

Comment:
Since we had adopted pytest style as our official way of writing test, would be better to keep all the tests in the same style.

```suggestion
class TestBakerAllowsSaveParameters:
```

Reply:
I agree that sticking with pytest is the way to go, but here's the thing: pytest do not support . That's why we had to use django's `TestCase` here.